cmd/ocitools/generate.go

```diff
@@ -83,19 +87,30 @@ var generateCommand = cli.Command{
 			return err
 		}
 
+		specgen.ExportOpts.Seccomp = context.Bool("seccomp-only")
+
 		if context.IsSet("output") {
-			output := context.String("output")
-			err = specgen.SaveToFile(output)
+			specgen.ExportOpts.Output = context.String("output")
 		} else {
-			err = specgen.Save(os.Stdout)
+			specgen.ExportOpts.Output = "STDOUT"
```

Comment:
On Fri, Jul 22, 2016 at 12:09:48PM -0700, Grant Seltzer Richman wrote:
> >  		} else {
> > -			err = specgen.Save(os.Stdout)
> > +			specgen.ExportOpts.Output = "STDOUT"
>
> Without having a function like `exportConfig` you would have to have
> the check for stdout/path inside both the if and the else, no?

I don't understand.  [1] only checks the output path if
context.IsSet("output"), not in the else branch.

[1]: https://github.com/opencontainers/ocitools/pull/128/files#r71928565